docs/Integrating-with-Finch/Integrate-Finch-Connect/Embed-Connect.md

```diff
@@ -19,34 +20,40 @@ type: tab
 title: React SDK
 -->
 
-To open Finch Connect with React, you will need to instantiate the [React SDK](https://github.com/Finch-API/react-finch-connect) with the `useFinchConnect` hook and invoke the returned `open` method to display the view for your user.
+To open Finch Connect with React, you will need to instantiate the [React SDK](https://github.com/Finch-API/react-connect) with the `useFinchConnect` hook and invoke the returned `open` method to display the view for your user.
 
 ### 1. Install the React Finch Connect SDK
 
-Inside your code base folder, use the command line to install `react-finch-connect`
+Inside your code base folder, use the command line to install `react-connect`
```

Comment:
Lines 27 to 37 are the only ones relevant for this PR. The rest of the changes are just prettier doing a better formatting for the markdown file on save.